JsonFlatFileDataStore/DataStore.cs

```diff
@@ -28,11 +28,13 @@ public class DataStore : IDataStore
         private readonly CancellationTokenSource _cts = new CancellationTokenSource();
         private readonly ExpandoObjectConverter _converter = new ExpandoObjectConverter();
         private readonly JsonSerializerSettings _serializerSettings = new JsonSerializerSettings() { ContractResolver = new CamelCasePropertyNamesContractResolver() };
+        private readonly Aes256 _aes256;
```

Comment:
This would follow style better that has been used in `DataStore`

```
private readonly Func<string, string> _encryptJson;
private readonly Func<string, string> _decryptJson;

if (string.IsNullOrWhiteSpace(encryptionKey))
{
    _encryptJson = new Func<string, string>(json => json);
    _decryptJson = new Func<string, string>(json => json);
}
else
{
    var aes256 = new Aes256();
    _encryptJson = new Func<string, string>(json => aes256.Encrypt(json, encryptionKey));
    _decryptJson = new Func<string, string>(json => aes256.Decrypt(json, encryptionKey));
}
```

Reply:
Agreed, looks much better, PR is now updated with these changes